src/Controller/ExportController.php

```diff
@@ -296,7 +303,7 @@ private function buildXmlProducts(
                     $productEntity,
                     $this->router,
                     $this->container,
-                    $salesChannelContext->getContext(),
+                    $this->salesChannelContext->getContext(),
```

Comment:
You can just use `$this->salesChannelContext` as a parameter, because the `$context` is never used in `FindologicProduct`.
Doing that, you can remove the `fin_search.sales_channel_context` service afterwards.

Reply:
No because that would break backwards compatibility.